src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+.article {
+  font-size: 13px;
+  margin: 0;
+}
+
+.h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+.h2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
```

Comment:
class names that equal to tag names make no sense
The name of the element class should make a sense. It should describe what the role of the element is on this page or describe its state.